master/static/srv/get_experiments.sql

```diff
@@ -28,9 +33,10 @@ WITH filtered_exps AS (
                     ELSE e.config->'labels' END
                 ))
             )
-        AND ($5 = '' OR POSITION($5 IN (e.config->>'description')) > 0)
+        AND ($5 = '' OR (e.config->>'description') ILIKE  ('%%' || $5 || '%%'))
```

Comment:
what is %% (i'm only aware of % and i dont think doubling what i know would change behavior)?

Reply:
I needed to escape it otherwise it'd get substituted and it's because of sprintf there

```
func (db *PgDB) QueryProtof(
	queryName string, args []interface{}, v interface{}, params ...interface{}) error {
	query := db.queries.getOrLoad(queryName)
	if len(args) > 0 {
		query = fmt.Sprintf(query, args...)
	}

```

 hence escaping it with go syntax otherwise it'd interfere with the orderby string. 
related to args vs params substitution in our 

` select  ('%' || 'asdf' || '%' ) as test from users where username LIKE ('%' || ''|| '%');` works fine in pgsql but through our queryproto:

`determined-db_1       | 2021-05-26 21:58:17.026 UTC [216] STATEMENT:  select  ('%!'(string=id ASC) || 'asdf' || '%!'(MISSING) ) as test from users where username LIKE ('%!'(MISSING) || ''|| '%!'(MISSING))`

Reply:
oh yeah, duh, query proto _f_. ok, i was just curious if this was some new thing I didn't know lol